.husky/.gitignore

```diff
@@ -0,0 +1 @@
+_
```

Comment:
What's the purpose of this file? Can the root `.gitignore` be used instead?

Reply:
there's a `_` folder generated under `.husky/`, which should be ignored.

This file is auto-generated by `npx mrm@2 lint-staged` https://prettier.io/docs/en/precommit.html#option-1-lint-stagedhttpsgithubcomokonetlint-staged